src/Wallet.ts

```diff
@@ -24,23 +24,23 @@ export type WalletParams = {
  * @property {RPC} RPC - Main RPC Handler Instance
  * @property {UtilityCollection} Utils - Utility Collection
  */
-class Wallet {
-    private chainId: Number;
+export default class Wallet {
+    public chainId: Number;
     public Account: any;
-    private Transaction: any;
-    private Rpc: any;
-    private Utils: any;
+    public Transaction: any;
+    public Rpc: any;
+    public Utils: any;
```

Comment:
If you need to access these for tests see @jona 's insight here. He initially made this private which I was okay with -- We could add getters for the testing suite so we don't have random intellisense suggestions for what should probably be private scoped variables of these class. Thoughts?